helm/managed/logging/elastic/templates/NOTES.txt

```diff
@@ -0,0 +1,10 @@
+Thank you for installing {{ .Chart.Name }}.
+A lightweight Elastic Search instance for HPCC component log processing.
+
+This deployment varies slightly from defaults set by Elastic, please review the effective values.
+
+Of note, Elastic Search declares PVC which might require explicit manual removal when no longer needed.
+
+  elasticsearch
+    volumeClaimTemplate
+{{ toYaml .Values.elasticsearch.volumeClaimTemplate | indent 6 }}
```

Comment:
this won't tell you the PVC I think?

Reply:
there may be no way to do so

Reply:
Correct, and this blurb might not be helpful. Here's a sample output:
  _elasticsearch
    volumeClaimTemplate
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 5Gi_
I might remove it and focus on the warning above it